File: snippets/python_snippets/static_channel_backup.py
import breez_sdk
import logging

sdk_services = breez_sdk.BlockingBreezServices


def retrieve_backup_files():
    try:
        # ANCHOR: static-channel-backup
        req = breez_sdk.StaticBackupRequest(working_dir="<working directory>")
        backup_data = breez_sdk.static_backup(req)
        # ANCHOR_END: static-channel-backup
        return backup_data
    except Exception as error:
        logging.error(error)
        raise

File: snippets/kotlin_mpp_lib/shared/src/commonMain/kotlin/com/example/kotlinmpplib/StaticChannelBackup.kt
package com.example.kotlinmpplib

import breez_sdk.*
class StaticChannelBackup {
    fun backup(sdk: BlockingBreezServices) {
        // ANCHOR: static-channel-backup
        try {
            val backupData = staticBackup(StaticBackupRequest("<working directory>"))
        } catch (e: Exception) {
            // handle error
        }
        // ANCHOR_END: static-channel-backup
    }
}